daemon/main.go

```diff
@@ -408,6 +408,9 @@ func init() {
 	flags.Bool("sidecar-http-proxy", false, "Disable host HTTP proxy, assuming proxies in sidecar containers")
 	flags.MarkHidden("sidecar-http-proxy")
 	viper.BindEnv("sidecar-http-proxy", "CILIUM_SIDECAR_HTTP_PROXY")
+	flags.String("sidecar-istio-proxy-image", "cilium/istio_proxy",
```

Comment:
shouldn't this be `docker.io/cilium/istio_proxy`? To be honest this should be image sha256

Reply:
It’s a regexp. I think this is sufficient to avoid matching the wrong images, for example Istio’s vanilla images. That was my intent.

Reply:
@aanm I'm not sure I fully understand your concern. This matches the image string of a container with the name "istio-proxy" inside of a pod. It's very specific.

Reply:
ok... I think I got it.